sdk/storage/azure-storage-queue/src/test/resources/session-records/AzuriteTests.azuriteUrlConstructingQueueClientWithConnectionString[1].json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Should these be marked @DoNotRecord?

Reply:
I'm going to leave that for now as this is just a one-to-one port